server/proxy/pf_context.c

```diff
@@ -77,5 +77,45 @@ rdpContext* p_client_context_create(rdpSettings* clientSettings,
 	settings->ServerPort = port;
 	settings->SoftwareGdi = FALSE;
 	settings->RedirectClipboard = FALSE;
+	/* Client Monitor Data */
+	settings->MonitorCount = clientSettings->MonitorCount;
```

Comment:
maybe we can move the monitor stuff to its own func.

Reply:
or to` pf_common_copy_settings`

Reply:
not get your reasoning to move to common settings, as those settings are one-sided (only client settings). i'd like not to change it until will open a PR that uses `freerdp_settings_clone`, then we will remove it